### PR TITLE
Copyright corrections

### PR DESCRIPTION
**Pull request by yaminikb**

### CODE REVIEW

**Comment by nigeldeakin** [comment]
Fine by me.

**Comment by nigeldeakin** [comment]
I'm unable to comment on whether this is the right legal text, but I have no objections to making changes here.

File: samples/tibcosample/docroot/index.html [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+    EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+    PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+    PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+    LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+    NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+    SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+-->
+
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```
Comment (bshannon) [inline samples/tibcosample/docroot/index.html]: The DOCTYPE should be above the comment.

File: seebeyondtests/build.xml [hunk 2]
```diff
@@ -1,25 +1,41 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-
 <!DOCTYPE project [
```
Comment (bshannon) [inline seebeyondtests/build.xml]: I don't think the comment can come in the middle of the DOCTYPE.  Try combining the DOCTYPE into one line, even though it won't look as nice.  That way the copyright plugin won't mess with it.

File: seebeyondtests/stress.dmq/build.xml [hunk 3]
```diff
@@ -1,26 +1,42 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-
 <!DOCTYPE project [
```
Comment (bshannon) [inline seebeyondtests/stress.dmq/build.xml]: As above, don't split the DOCTYPE.

File: seebeyondtests/stress.lookup/build.xml [hunk 4]
```diff
@@ -1,26 +1,42 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-
 <!DOCTYPE project [
```
Comment (bshannon) [inline seebeyondtests/stress.lookup/build.xml]: As above, don't split the DOCTYPE.

File: seebeyondtests/stress.topic.clientId/build.xml [hunk 5]
```diff
@@ -1,26 +1,42 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-
 <!DOCTYPE project [
```
Comment (bshannon) [inline seebeyondtests/stress.topic.clientId/build.xml]: As above, don't split the DOCTYPE.

File: seebeyondtests/stress.topic/build.xml [hunk 6]
```diff
@@ -1,26 +1,42 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-
 <!DOCTYPE project [
```
Comment (bshannon) [inline seebeyondtests/stress.topic/build.xml]: As above, don't split the DOCTYPE.

**Pull request merged**